balancer-js/src/modules/data/pool/onChainData.ts

```diff
@@ -0,0 +1,277 @@
+import { formatFixed } from '@ethersproject/bignumber';
```

Comment:
Should be able to delete this file? Everything should use `balancer-js/src/modules/sor/pool-data/onChainData.ts`